pkl-gradle/src/test/kotlin/org/pkl/gradle/JavaCodeGeneratorsTest.kt

```diff
@@ -127,33 +113,24 @@ class JavaCodeGeneratorsTest : AbstractTest() {
             paramsAnnotation = "javax.inject.Named"
             nonNullAnnotation = "javax.annotation.Nonnull"
             settingsModule = "pkl:settings"
+            packageMapping = [
+              'org': 'foo.bar'
+            ]
           }
         }
       }
     """
     )
   }
 
-  private fun writeGradlePropertiesFile() {
-    writeFile("gradle.properties", """
-      systemProp.http.proxyHost=proxy.config.pcp.local
-      systemProp.http.proxyPort=3128
-      systemProp.http.nonProxyHosts=localhost|*.apple.com
-      
-      systemProp.https.proxyHost=proxy.config.pcp.local
-      systemProp.https.proxyPort=3128
-      systemProp.https.nonProxyHosts=localhost|*.apple.com
-    """)
-  }
-  
   private fun writePklFile() {
     writeFile(
       "mod.pkl", """
         module org.mod
   
         class Person {
           name: String
-          addresses: List<Address>
+          addresses: List<Address?>
```

Comment:
Why was this changed?

Reply:
Previously, there was two almost identical Pkl scripts here, whose difference was basically in this `?`. The question mark was necessary, because apparently the code generated without this question mark fails to compile with some really strange error about annotations. I unified these two scripts to avoid repetition, and to make sure that the code can be compiled, I switched it to be nullable. I think it does not matter much here because we are not really concerned with the actual contents of the generated classes, we check that the Gradle plugin works and can be used to generate and compile code properly.

Reply:
Ah, probably because the in-memory compiler didn't have access to our NonNull annotation. Gotcha, thanks for the explanation.